repo/packages/C/chronos/0/config.json

```diff
@@ -1 +1,200 @@
-{}
+{
+  "type": "object",
+  "properties": {
+    "assets_path": {
```

Comment:
_All_ of these property names should be namespaced to the package.  E.g. `chronos/assets_path`.  See the Marathon example and try to conform to the precedent set there.